go/router/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fasthttp/router"
+	"github.com/valyala/fasthttp"
+)
+
+func ShowEmpty(ctx *fasthttp.RequestCtx) {
+	ctx.WriteString("")
+}
+
+func ShowID(ctx *fasthttp.RequestCtx) {
+	fmt.Fprintf(ctx, "%s", ctx.UserValue("id"))
```

Comment:
You could write in the response body with `ctx.WriteString(ctx.UserValue("id").(string))`. I think it's faster than `fmt.Fprintf(ctx, ....)`